src/hexer_docs.erl

```diff
@@ -0,0 +1,106 @@
+-module(hexer_docs).
+
+-export([publish/0]).
+
+%%------------------------------------------------------------------------------
+%% API
+%%------------------------------------------------------------------------------
+
+-spec publish() -> ok | {error, any()}.
+publish() ->
+  hexer_utils:print("Publishing..."),
+  %% We need the following information to build the package:
```

Comment:
The code that comes after this comment doesn't seem to reflect the contents of the comment. I bet the comment is outdated. I would just remove it.